### PR TITLE
Indicium blog post

### PR DESCRIPTION
**Pull request by gsathya**

### CODE REVIEW

**Comment by mathiasbynens** [comment]
@RReverser @surma Are you on top of this?

**Comment by zeynepCankara** [comment]
@googlebot I consent.

**Review by camillobruni — APPROVED** [review]
LGTM

**Review by camillobruni — APPROVED** [review]

File: src/blog/system-analyzer.md
Comment (rmcilroy) [inline src/blog/system-analyzer.md:116]: in the previously linked > in a previous

File: src/blog/system-analyzer.md
Comment (rmcilroy) [inline src/blog/system-analyzer.md:118]: we toggle -> . To do so, we can

File: src/blog/system-analyzer.md
Comment (rmcilroy) [inline src/blog/system-analyzer.md:126]: The text in this image: "cause dynamically adding property" -> "causes the addition of a dynamically added property"

File: src/blog/system-analyzer.md
Comment (rmcilroy) [inline src/blog/system-analyzer.md:130]: I'd drop this paragraph and just add " and can use this insight to address the performance regression" to the sentence above.

File: src/blog/system-analyzer.md
Comment (rmcilroy) [inline src/blog/system-analyzer.md:132]: "As a common advice we say" -> "In general it is sound advice"

File: src/blog/system-analyzer.md
Comment (rmcilroy) [inline src/blog/system-analyzer.md:150]: "we see on the that execution of the two snippets we defined at the beginning of the case study perform very similarly." -> "we see that the execution of the two snippets defined at the beginning of the case study perform very similarly."

File: src/blog/system-analyzer.md
Comment (rmcilroy) [inline src/blog/system-analyzer.md:152]: I'd drop this graph

File: src/blog/system-analyzer.md
Comment (rmcilroy) [inline src/blog/system-analyzer.md:156]: Move this above the image

Comment (rmcilroy) [reply src/blog/system-analyzer.md:156]: In fact, I'd just keep "In an updated trace, we see that the polymorphic IC state is avoided as we are not creating multiple maps for the same type of objects." at the end of the paragraph above.

File: src/blog/system-analyzer.md
Comment (rmcilroy) [inline src/blog/system-analyzer.md:178]: searched through -> searched for

File: src/blog/system-analyzer.md
Comment (rmcilroy) [inline src/blog/system-analyzer.md:14]: Past three -> The past three

File: src/blog/system-analyzer.md
Comment (rmcilroy) [inline src/blog/system-analyzer.md:18]: Previous -> However, previous

File: src/blog/system-analyzer.md
Comment (gsathya) [reply src/blog/system-analyzer.md:118]: Done

File: src/blog/system-analyzer.md
Comment (gsathya) [reply src/blog/system-analyzer.md:130]: Done

File: src/blog/system-analyzer.md
Comment (gsathya) [reply src/blog/system-analyzer.md:132]: Done

File: src/blog/system-analyzer.md
Comment (gsathya) [reply src/blog/system-analyzer.md:150]: Done

File: src/blog/system-analyzer.md
Comment (gsathya) [reply src/blog/system-analyzer.md:156]: Done

File: src/blog/system-analyzer.md
Comment (gsathya) [reply src/blog/system-analyzer.md:178]: Done

File: src/blog/system-analyzer.md
Comment (gsathya) [reply src/blog/system-analyzer.md:18]: Done

File: src/blog/system-analyzer.md
Comment (gsathya) [reply src/blog/system-analyzer.md:152]: Done

File: src/blog/system-analyzer.md
Comment (gsathya) [reply src/blog/system-analyzer.md:14]: Done

File: src/blog/system-analyzer.md
Comment (gsathya) [reply src/blog/system-analyzer.md:126]: Images can't be edited anymore :(

Gonna leave this is as such as I don't want to delete it

File: src/blog/system-analyzer.md
Comment (gsathya) [reply src/blog/system-analyzer.md:116]: removed this line entirely

**Comment by RReverser** [comment]
Can you recompress images with https://squoosh.app/ or command-line OptiPNG / OxiPNG? E.g. trying case1_1.png got me from 194 KB to 167 KB, I imagine others will give similar savings.

**Comment by gsathya** [comment]
> Can you recompress images with https://squoosh.app/ or command-line OptiPNG / OxiPNG? E.g. trying case1_1.png got me from 194 KB to 167 KB, I imagine others will give similar savings.

```
** Processing: src/_img/system-analyzer/case1_1.png
954x706 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 167853 bytes
Input file size = 168372 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 167853

src/_img/system-analyzer/case1_1.png is already optimized.

** Processing: src/_img/system-analyzer/case1_2.png
960x720 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 154661 bytes
Input file size = 154766 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 154661

Output IDAT size = 154661 bytes (no change)
Output file size = 154718 bytes (48 bytes = 0.03% decrease)

** Processing: src/_img/system-analyzer/case1_3.png
960x720 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 166588 bytes
Input file size = 166705 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 166588

Output IDAT size = 166588 bytes (no change)
Output file size = 166645 bytes (60 bytes = 0.04% decrease)

** Processing: src/_img/system-analyzer/case1_4.png
960x720 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 167976 bytes
Input file size = 168093 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 167976

Output IDAT size = 167976 bytes (no change)
Output file size = 168033 bytes (60 bytes = 0.04% decrease)

** Processing: src/_img/system-analyzer/case2_1.png
960x720 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 163427 bytes
Input file size = 163532 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 163427

Output IDAT size = 163427 bytes (no change)
Output file size = 163484 bytes (48 bytes = 0.03% decrease)

** Processing: src/_img/system-analyzer/ic-panel2.png
960x720 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 138364 bytes
Input file size = 138469 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 138364

Output IDAT size = 138364 bytes (no change)
Output file size = 138421 bytes (48 bytes = 0.03% decrease)

** Processing: src/_img/system-analyzer/ic-panel3.png
960x720 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 98924 bytes
Input file size = 99017 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 98924

Output IDAT size = 98924 bytes (no change)
Output file size = 98981 bytes (36 bytes = 0.04% decrease)

** Processing: src/_img/system-analyzer/ic-panel4.png
960x720 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 101912 bytes
Input file size = 102005 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 101912

Output IDAT size = 101912 bytes (no change)
Output file size = 101969 bytes (36 bytes = 0.04% decrease)

** Processing: src/_img/system-analyzer/ic-panel.png
960x720 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 133895 bytes
Input file size = 134000 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 133895
  zc = 9  zm = 8  zs = 0  f = 4         IDAT size = 133812
  zc = 9  zm = 8  zs = 0  f = 5         IDAT size = 133493

Selecting parameters:
  zc = 9  zm = 8  zs = 0  f = 5         IDAT size = 133493

Output IDAT size = 133493 bytes (402 bytes decrease)
Output file size = 133550 bytes (450 bytes = 0.34% decrease)

** Processing: src/_img/system-analyzer/indicium-logo.png
122x145 pixels, 4x8 bits/pixel, RGB+alpha
Reducing image to 3x8 bits/pixel, RGB
Input IDAT size = 27109 bytes
Input file size = 27857 bytes

Trying:
  zc = 9  zm = 9  zs = 0  f = 0         IDAT size = 24144
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 24122
  zc = 9  zm = 9  zs = 0  f = 1         IDAT size = 22774
  zc = 9  zm = 9  zs = 1  f = 1         IDAT size = 21890
  zc = 9  zm = 9  zs = 1  f = 2         IDAT size = 21352
  zc = 9  zm = 9  zs = 1  f = 4         IDAT size = 21042
  zc = 9  zm = 9  zs = 1  f = 5         IDAT size = 20682

Selecting parameters:
  zc = 9  zm = 9  zs = 1  f = 5         IDAT size = 20682

Output IDAT size = 20682 bytes (6427 bytes decrease)
Output file size = 21418 bytes (6439 bytes = 23.11% decrease)

** Processing: src/_img/system-analyzer/initial-program-performance.png
600x371 pixels, 4x8 bits/pixel, RGB+alpha
Reducing image to 8 bits/pixel, 206 colors in palette
Input IDAT size = 8096 bytes
Input file size = 8153 bytes

Trying:
  zc = 9  zm = 9  zs = 0  f = 0         IDAT size = 3711
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 3711

Selecting parameters:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 3711

Output IDAT size = 3711 bytes (4385 bytes decrease)
Output file size = 4398 bytes (3755 bytes = 46.06% decrease)

** Processing: src/_img/system-analyzer/map-panel.png
960x720 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 157908 bytes
Input file size = 158013 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 157908

Output IDAT size = 157908 bytes (no change)
Output file size = 157965 bytes (48 bytes = 0.03% decrease)

** Processing: src/_img/system-analyzer/performance-graph.png
600x371 pixels, 4x8 bits/pixel, RGB+alpha
Reducing image to 3x8 bits/pixel, RGB
Input IDAT size = 10338 bytes
Input file size = 10395 bytes

Trying:
  zc = 9  zm = 9  zs = 0  f = 0         IDAT size = 9448
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 9448

Selecting parameters:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 9448

Output IDAT size = 9448 bytes (890 bytes decrease)
Output file size = 9505 bytes (890 bytes = 8.56% decrease)

** Processing: src/_img/system-analyzer/source-panel.png
960x720 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 92642 bytes
Input file size = 92723 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 92642

Output IDAT size = 92642 bytes (no change)
Output file size = 92699 bytes (24 bytes = 0.03% decrease)

** Processing: src/_img/system-analyzer/stats-panel.png
960x720 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 147396 bytes
Input file size = 147501 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 147396

Output IDAT size = 147396 bytes (no change)
Output file size = 147453 bytes (48 bytes = 0.03% decrease)

** Processing: src/_img/system-analyzer/system-analyzer-overview.png
960x720 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 177958 bytes
Input file size = 178075 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 177958

Output IDAT size = 177958 bytes (no change)
Output file size = 178015 bytes (60 bytes = 0.03% decrease)

** Processing: src/_img/system-analyzer/timeline-panel2.png
960x720 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 168643 bytes
Input file size = 168760 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 168643

Output IDAT size = 168643 bytes (no change)
Output file size = 168700 bytes (60 bytes = 0.04% decrease)

** Processing: src/_img/system-analyzer/timeline-panel.png
960x720 pixels, 4x8 bits/pixel, RGB+alpha
Input IDAT size = 169917 bytes
Input file size = 170034 bytes

Trying:
  zc = 9  zm = 8  zs = 0  f = 0         IDAT size = 169917

Output IDAT size = 169917 bytes (no change)
Output file size = 169974 bytes (60 bytes = 0.04% decrease)
```

**Review by RReverser — APPROVED** [review]
LGTM, thanks.

File: src/blog/system-analyzer.md
Comment (RReverser) [inline src/blog/system-analyzer.md:6]: @gsathya Please update the date to today.

**Comment by RReverser** [comment]
Wait, no, don't merge this yet! We have specific times when we publish :)

**Comment by gsathya** [comment]
> Wait, no, don't merge this yet! We have specific times when we publish :)

Oh no. Should I revert it?

**Comment by RReverser** [comment]
> Oh no. Should I revert it?

I'll do it.

**Comment by RReverser** [comment]
Done + restored the branch. Can you open a new PR? (I don't think it's possible to reopen)

**Comment by gsathya** [comment]
> Done + restored the branch. Can you open a new PR? (I don't think it's possible to reopen)

https://github.com/v8/v8.dev/pull/486

**Pull request merged**
